### DISCUSSION TITLE
How to use custom css in project setting?

### DISCUSSION THREAD

**Discussion by khunsatant** [directus/directus#3902] [Archive]
Hi Guys,
I try to custom the css in Project Setting > App Override. I put the custom variable as below but nothing happen,
`:root {
	--input-padding: 5px;
}`
Note: default value is 12px;
Do i put the wrong syntax? pls suggest.
Thanks.

**Comment by benhaynes** [comment]
@rijkvanzanten @philleepflorence — Another for the Docs!

**Comment by rijkvanzanten** [comment]
@khunsatant I think it's a scoping issue, the Directus app has precedence, so your change isn't applied. Try:

```css
#app {
  --input-padding: 5px;
}
```

🙂

**Reply by khunsatant** [reply]
Thank you for your quick response but it's doesn't work for me.
but I found the solution might be help others.
<pre>
:root{
    --form-column-max-width: 100% !important;
}
.content .v-form{
    --input-height: 50px !important;
    --input-padding: 5px 16px !important;
    --form-vertical-gap: 20px !important;   
}
</pre>
